commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/ftp/FtpFileSystemConfigBuilder.java

```diff
@@ -244,6 +245,16 @@ public Boolean getPassiveMode(final FileSystemOptions options) {
         return getBoolean(options, PASSIVE_MODE);
     }
 
+    /**
+     * Gets the active port range.
+     *
+     * @param options The FileSystemOptions.
+     * @return the FTP active port range
```

Comment:
New public methods need a 'since' tag.

Reply:
what is the value of since tag should i use ?

Reply:
The file pom.xml says '<version>2.10.0-SNAPSHOT</version>' so use '2.10.0'

Reply:
since tag is added